runtime/doc/windows.txt

```diff
@@ -1136,6 +1136,8 @@ list of buffers. |unlisted-buffer|
 		the current buffer remains being edited.  See |:buffer-!| for
 		[!].  This will also edit a buffer that is not in the buffer
 		list, without setting the 'buflisted' flag.
+		The notation with single quotes does not work here,
+		`:buf 12'345'` uses 12'345 as a buffer name.
```

Comment:
Feels scriptversion4 & vim9scriptey. I don't think we include this:
https://github.com/vim/vim/commit/af377e34b01ba00f9520d2b9de1f911e72db0114